File: src/php/240324.php
<?php
$nums = [1,3,4,2,2];
$dup = getDuplicate($nums);
print_r($dup);
exit;
function getDuplicate($nums){
    $hashMap = [];
    foreach($nums as $num){
        if(isset($hashMap[$num])){
            return $num;
        }
        else{
            $hashMap[$num] = 1;
        }
    }
}

File: src/python/240324.py
def findDuplicates(nums):
    """
    :type nums: List[int]
    :rtype: List[int]
    """
    hashMap = {}
    for num in nums:
        if num in hashMap:
            return num
        else:
            hashMap[num] = 1
nums = [1,3,4,2,2]
res = findDuplicates(nums)
print(res)
